service-templates/apigw-lambda-svc/v1/instance_infrastructure/README.md

```diff
@@ -0,0 +1,78 @@
+# Instance
+
+## Local Development & Testing
```

Comment:
Just brief explanation of local setup and using `*.tfvars` file for local dev/testing